sdk/core/System.ClientModel/src/Convenience/ClientResult.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Internal;
+using System.ClientModel.Primitives;
+
+namespace System.ClientModel;
+
+public class ClientResult
```

Comment:
Where are the doc comments for the public members?  Without them, our MS Docs pages will be bare.

Reply:
I have been tracking these with:
- https://github.com/Azure/azure-sdk-for-net/issues/40758
- https://github.com/Azure/azure-sdk-for-net/issues/41044

since the APIs are still in flux to conserve time possibly documenting irrelevant APIs.

But, if the group feels they are a criterion for merging to main, I will add them to this PR.  Let me know!

Reply:
No objection to delaying until there is stability, so long as we have it as a pre-release criteria.

Reply:
Yep, I have both those issues marked as release blockers.

Reply:
... and slated for the "release activities" in the post-RC/pre-GA milestone.